### PR TITLE
Feature chain helpers utils added

### PR DESCRIPTION
**Pull request by satanworker**
Feature chain utils added

### CODE REVIEW

File: src/utils/chainInfoHelpers.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  }>((accumulator, chainId) => {
+    const numberChainId = Number(chainId);
+    const providerInstance = {
+      get instance() {
+        if (initalizedProviders[numberChainId]) {
+          return initalizedProviders[numberChainId];
+        } else {
+          // TODO: add fallback provider
+          initalizedProviders[numberChainId] =
+            new providers.JsonRpcBatchProvider(urls[numberChainId][0]);
```
Comment (kyzia551) [inline src/utils/chainInfoHelpers.ts:56]: shouldn't it be return here?

File: src/utils/chainInfoHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      },
+    } as {
+      instance: providers.JsonRpcBatchProvider;
+    };
+
+    accumulator[numberChainId] = providerInstance;
+    return accumulator;
+  }, {});
+
+  function getAddChainParameters(
```
Comment (kyzia551) [inline src/utils/chainInfoHelpers.ts:67]: not sure that **Get** and **Add** should be at the same time in the function name :)

File: src/utils/chainInfoHelpers.ts (hunk 1 above)
Comment (satanworker) [reply src/utils/chainInfoHelpers.ts:56]: yes, will fix

File: src/utils/chainInfoHelpers.ts (hunk 2 above)
Comment (satanworker) [reply src/utils/chainInfoHelpers.ts:67]: changerd

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**